### PR TITLE
Restored case finding mobile v1

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

File: custom/enikshay/ucr/reports/case_finding_mobile_v2.json [hunk 1]
```diff
@@ -0,0 +1,416 @@
+{
```
Comment (kkrampa) [inline custom/enikshay/ucr/reports/case_finding_mobile_v2.json]: Intentionally left and not added to settings

Comment (czue) [inline custom/enikshay/ucr/reports/case_finding_mobile_v2.json]: why?

Comment (kkrampa) [inline custom/enikshay/ucr/reports/case_finding_mobile_v2.json]: I guess they want to use this in further version of app but I can remove this if you want.

Comment (czue) [inline custom/enikshay/ucr/reports/case_finding_mobile_v2.json]: no it's fine, was just curious.

**Review by czue — APPROVED** [review]

**Pull request merged**